### ISSUE TITLE
Is is possible to have a web or iOS client to decode and receive files?

### ISSUE THREAD

**Issue by troilus** [sz3/cfc#33]

**Comment by sz3** [comment]
Not currently. I've been thinking about writing a web app for it though.

**Comment by gaowanliang** [comment]
I've been following this project lately and find it interesting, I'll be spending some time writing a client in Flutter that will support Android, iOS, Linux, macOS and Windows platforms, as well as redesigning the site using Material Design 3, the current site looks a bit spartan.

**Comment by sz3** [comment]
I wrote up thoughts on the approach *I* hope to take here: https://github.com/sz3/libcimbar/issues/71#issuecomment-2337147651

But other R&D attempts at frontends are welcome. Ideally the libcimbar component will evolve into something that can act as a native interface accepting a sequence of (e.g.) rgb/bgr buffers and either returning a final byte buffer on completed transfer (this seems more flexible), or writing the file itself. The UI components can then concentrate -- hopefully -- on UI things.

**Comment by iuvi7** [comment]
> edit: I can't read, bear with me as I edit this into something sensible 😆
> 
> It should be possible. I've been thinking (quite a lot) about writing a web app for the decoder.

Yes **_PLEASE_**! I found it easy to send by airgap from my Notebook to my Android, but i can not find any solution to screen-cast QR's as files **_from Android to Notebook_**, so it can works both directions. As well as offline version with ,js (maybe integrate scanner into html?

Thanks :)
